.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Feature Request
+about: Make suggestions on how TM:PE could improve.
+title: ''
+labels: 'feature'
+assignees: ''
+---
+
+<!-- Before submitting this feature, please search and make sure it hasn't already been suggested, thanks! -->
+
+### Description
+<!-- In a few sentences, summarize what your idea is. -->
+
+
+### Sources / Links
+<!-- Post any links below that describe how your issue works. For example, with turn on red it'd be the link to a "turn on red rule" in a driver's handbook; things like that. -->
+
+
+### Screenshots
+<!-- If you have screenshots showing how this feature works, put them below. How to: https://bit.ly/2Kc8owO -->
+
+
+### Additional Info
+<!-- Any other things worthy of being noted? Let us know what they are below. -->
+
+
+<!--
+EXAMPLE:
+
+### Description
+<!-- In a few sentences, summarize what your idea is. ->
+Right on red (or left on red if map in left hand drive mode) should allow vehicles to turn at a red light after stopping.
+
+### Sources / Links
+<!-- Post any links below that describe how your issue works. For example, with turn on red it'd be the link to a "turn on red rule" in a driver's handbook; things like that. ->
+https://dld.utah.gov/wp-content/uploads/sites/17/2018/12/Driver-Handbook-2018-2019.pdf#page=29
+
+### Screenshots
+<!-- If you have screenshots showing how this feature works, put them below. How to: https://bit.ly/2Kc8owO ->
+![image](https://user-images.githubusercontent.com/9030085/44266955-c67dc180-a26f-11e8-94e4-35e18dc05e3d.png)
```

Comment:
we're getting in to realms of teaching people github while they are trying to make a feature request

suggest having a read of: https://alistapart.com/article/flywheelsandfriction :)